src/snapshot.js

```diff
@@ -69,6 +69,31 @@ export function getPlayerSnapshot(player) {
   }
   snapshotObject.suppressedTracks = suppressedTracks;
 
+  /**
+   * iOS Safari will change caption mode to 'showing' if a user previously
+   * turned captions on manually for that video source, so this TextTrackList
+   * 'change' event handler will re-disable them in case that occurs during ad playback
+   */
+  const iOSTrackListChangeHandler = function() {
+    if (player.ads.isAdPlaying()) {
+      const textTrackList = player.textTracks();
+
+      for (let i = 0; i < textTrackList.length; i++) {
+        const track = textTrackList[i];
+
+        if (track.mode === 'showing') {
+          track.mode = 'disabled';
+        }
+      }
+    }
+  };
+
+  if (videojs.browser.IS_IOS && !Array.isArray(tracks)) {
```

Comment:
I think we talked about including `featuresNativeTextTracks ` in this conditional, how do you feel about that?